src/content/changelog/4-83-2-2021-01-29.md

```diff
@@ -0,0 +1,19 @@
+---
+title: '4.83.2: 2021-01-29'
+date: 2021-01-29T04:00:00.000Z
+version: 4.83.2
+changelog:
+  - API
+---
+
+### Changed
+
+- The limit of Firewalls per Linode has been increased from 3 to 5. Descriptions for Firewall ([/networking/firewalls](https://www.linode.com/docs/api/networking/)) beta endpoints were updated to reflect this increase.
```

Comment:
the link doesn't seem right. text points to one thing but href points to another

Reply:
nice catch, fixed